### PR TITLE
Overview: Added some reference applications section with images

### PR DESCRIPTION
**Pull request by mmaciola**
Added some reference applications section (links to nui-demo/wearable-samples) with images to attract third party developers to use NUI.

### CODE REVIEW

**Comment by mijin85cho** [comment]
@mmaciola The update is in html tag. Was there any reason that you didn't use markdown style table? If not, please change the table to markdown style.

File: docs/application/dotnet/guides/nui/overview.md [hunk 1]
```diff
@@ -20,6 +20,22 @@ NUI offers the following main features:
 -   Easy-to-use [animation framework](animation.md) which hides the complexity of the underlying three-dimensional math
 -   Keyboard, touch, and mouse handling
 
+Some reference applications:
+<table style="text-align:center;">
+  <tr>
+    <th><img src="https://github.com/dalihub/nui-demo/raw/master/wearable-samples/res/wapps.png"/></th>
+    <th><img src="https://github.com/dalihub/nui-demo/raw/master/wearable-samples/res/wgallery.png"/></th>
+    <th><img src="https://github.com/dalihub/nui-demo/raw/master/wearable-samples/res/wmessege.png"/></th>
+    <th></th>
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/overview.md:29]: There is no image for WHome?

File: docs/application/dotnet/guides/nui/overview.md [hunk 2]
```diff
@@ -20,6 +20,22 @@ NUI offers the following main features:
 -   Easy-to-use [animation framework](animation.md) which hides the complexity of the underlying three-dimensional math
 -   Keyboard, touch, and mouse handling
 
+Some reference applications:
+<table style="text-align:center;">
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/overview.md:24]: As @mijin85cho mentioned, we usually follow markdown tags, kindly let us know if there is any specific reason why you have used HTML tags?

You can refer this file for markdown tags for table.

https://raw.githubusercontent.com/Samsung/tizen-docs/master/docs/application/native/tutorials/details/app-filtering.md

File: docs/application/dotnet/guides/nui/overview.md [hunk 3]
```diff
@@ -20,6 +20,22 @@ NUI offers the following main features:
...
+  <tr>
+    <th><img src="https://github.com/dalihub/nui-demo/raw/master/wearable-samples/res/wapps.png"/></th>
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/overview.md:26]: @mijin85cho, these images are placed on an external Github repo, is it okay to link to an image outside?

**Review by annie-samsung — COMMENTED** [review]
Provided comments

File: docs/application/dotnet/guides/nui/overview.md (hunk 3 above)
Comment (mijin85cho) [reply docs/application/dotnet/guides/nui/overview.md:26]: The image is in https://github.com/dalihub/nui-demo/, and the repository seems to be managed by the people in Samsung in charge of graphics/DALi/NUI. So I think it is okay.

**Comment by mmaciola** [comment]
Please find changes: images was moved locally, to the repository. Table was changed to markdown style

File: docs/application/dotnet/guides/nui/overview.md [hunk 4]
```diff
@@ -20,6 +20,11 @@ NUI offers the following main features:
 -   Easy-to-use [animation framework](animation.md) which hides the complexity of the underlying three-dimensional math
 -   Keyboard, touch, and mouse handling
 
+Some reference applications:
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/overview.md:23]: Can you change this line to:
The following are reference applications:

Comment (annie-samsung) [reply docs/application/dotnet/guides/nui/overview.md]: @mmaciola, gentle reminder to fix the comment.

**Review by annie-samsung — APPROVED** [review]
Looks good to me, approving

**Review by mijin85cho — APPROVED** [review]
Thanks for the update and reviewing.

**Pull request merged**